src/utils/itemProperties.jsx

```diff
@@ -164,7 +164,7 @@ export async function unpackHelper(model, eltArr, serial, ifcToRowCb) {
           throw new Error('Array contains non-reference type')
         }
         const refId = stoi(p.value)
-        if (model.getItemProperties) {
+        if (model.getItemProperties && model.getItemProperties !== 'undfined') {
```

Comment:
Hey, this doesn't look right.  There's a typo in undefined, but moreover, if fixed the right side will always return true if it is evaluated.  Does this really fix the issue?

Reply:
not really sure how to reproduce.
will turn it to a draft for now.